ibis/backends/snowflake/tests/test_udf.py

```diff
@@ -92,3 +92,67 @@ def test_builtin_agg_udf(con):
         expected = c.exec_driver_sql(query).cursor.fetch_pandas_all()
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_xgboost_model(con):
+    import ibis
+    from ibis import _
+
+    @udf.scalar.pandas(
+        packages=("joblib", "xgboost"), imports=("@MODELS/model.joblib",)
+    )
+    def predict_price(
+        carat_scaled: float, cut_encoded: int, color_encoded: int, clarity_encoded: int
+    ) -> int:
+        import sys
+
+        import joblib
+        import pandas as pd
+
+        import_dir = sys._xoptions.get("snowflake_import_directory")
+        model = joblib.load(f"{import_dir}model.joblib")
+        df = pd.concat(
+            [carat_scaled, cut_encoded, color_encoded, clarity_encoded], axis=1
+        )
+        df.columns = ["CARAT_SCALED", "CUT_ENCODED", "COLOR_ENCODED", "CLARITY_ENCODED"]
```

Comment:
Not a huge fan of having to do this renaming, but I'm not entirely sure how it can be avoided.